app/lib/brexit_checker/actions.yaml

```diff
@@ -414,6 +430,151 @@ actions:
   audience: citizen
   grouping_criteria:
   - visiting-uk
+- id: S037
```

Comment:
The diffs gone a bit weird for S037 - S045.

Reply:
I think if you review per commit then it looks OK, because the rejig happened in one commit, and the updates happen in a subsequent one.